spec/frequency_table_spec.rb

```diff
@@ -7,109 +7,88 @@
   include_context "with tables for holdings"
 
   let(:ft) { described_class.new }
-  let(:ht1) {
-    build(
-      :ht_item,
-      :spm,
-      ocns: [5],
-      access: "deny",
-      rights: "ic",
-      collection_code: "MIU"
-    )
-  }
-  let(:ht2) {
-    build(
-      :ht_item,
-      :spm,
-      ocns: [5, 6],
-      access: "deny",
-      rights: "ic",
-      collection_code: "PU"
-    )
-  }
-
-  before(:each) do
-    Cluster.each(&:delete)
-    Cluster.create(ocns: ht2.ocns)
-    insert_htitem ht1
-    insert_htitem ht2
-  end
+  let(:umich_data) { {umich: {spm: {"1": 1}}} }
+  let(:upenn_data) { {upenn: {spm: {"1": 1}}} }
+  let(:ft_data) { umich_data.merge upenn_data }
```

Comment:
Doesn't look like you're using `ft_data` anywhere else. If not, I'd delete line 12 and change the next line to:

```
let(:ft_with_data) { described_class.new(data: umich_data.merge upenn_data) }
```